bin/import-wikidata

```diff
@@ -87,7 +87,7 @@ async def main(args):
     if cache:
         # Do not query for the IDs we already have in cache
         new_ids = set(ids).difference(cache.keys())
-        print(f'Found {len(ids):,} Wikidata IDs, {len(new_ids) - len(ids):,} in cache, '
+        print(f'Found {len(ids):,} Wikidata IDs, {abs(len(new_ids) - len(ids)):,} in cache, '
```

Comment:
that's not a noop -- it was a bug and `abs()` hides it :).  I have fixed it by changing the order of what is subtracted from what.